src/tags/image-content-digest.tag

```diff
@@ -0,0 +1,48 @@
+<!--
+ Copyright (C) 2019 Jakob Ackermann
```

Comment:
Hum, this should not be the name of the project author? 🤔

Reply:
The headers serves a notice in case the file is taken out of its context. From what I have seen in the wild it lists the authors of code contributions and it is updated as (larger) changes are applied.
It may contain individuals of refer to a group of authors, e.g. listed in an AUTHORS or CONTRIBUTORS at the top level.

some examples:
- https://github.com/go-gitea/gitea/blob/b660a732ae283d863636ead9cc1a365ce1c0edc1/models/action.go
- https://github.com/moby/moby/blob/master/AUTHORS
- run `$ grep -r . -I -i -e 'Copyright (c)' | sed -E 's#./([^/]+).+?:(.+)#\1:\2#' | sort | uniq` where your projects are stored. optionally add ` | grep '@' -m 2000`.
  ```
  socket.io:Copyright (c) XXX <XXX@gmail.com>
  c9:Copyright (C) XXX <XXX@gmail.com>
  ```
- https://github.com/search?l=C&q=Copyright+%28C%29+2019&type=Code

Reply:
Fine, l will create a CONTRIBUTORS file with some TEMPLATE in a near future :slightly_smiling_face: